content/blog/2019/05/2019-05-22-outreachy-audit-log-project.adoc

```diff
@@ -0,0 +1,33 @@
+---
+:layout: post
+:title: Audit Logging in Jenkins: An Outreachy Project
```

Comment:
I suspect the CI job failure is because the title does not want to include a ':' character.

Reply:
I'm going to try adding quotes around the title. If that doesn't work, this is the way to go ^

Reply:
Looks like it just needed proper YAML quoting.